### DISCUSSION TITLE
Electron-style container, menu bar icons, etc?

### DISCUSSION THREAD

**Discussion by benallfree** [garethgeorge/backrest#153] [Ideas]
This is a great project! What kinds of thoughts do you have about a more desktop-like experience? 

I'm on OS X using Arq and I'm changing to this. The only thing I'm missing is some kind of status icon/notification to tell me if there is a problem. 

Well, that, and a `launch at startup` type option.

**Comment by garethgeorge** [comment]
Hey, this is something I’ve definitely been thinking about re: packaging and want to experiment with when I have some bandwidth (or I’d be very open to taking as a contribution if anyone has experience in the space) 

re: launch at startup, the install.sh script creates a launchd agent which should keep backrest running in the background as a startup task. You can take a look at that script and see if it does what you need.
